src/components/views/rooms/MessageComposer.tsx

```diff
@@ -359,14 +362,21 @@ export class MessageComposer extends React.Component<IProps, IState> {
         });
     };
 
-    private onRichTextToggle = () => {
-        this.setState((state) => ({
-            isRichTextEnabled: !state.isRichTextEnabled,
-            initialComposerContent: !state.isRichTextEnabled
-                ? state.composerContent
-                : // TODO when available use rust model plain text
-                  htmlToPlainText(state.composerContent),
-        }));
+    private onRichTextToggle = async () => {
+        // due to this repo targetting ES6, we need to import dynamically
+        const conversionFunctions = await import("@matrix-org/matrix-wysiwyg");
```

Comment:
I think that will be a good idea to wrap this dynamic import in the RTE files.

You can take a look at the SendWysiwygComposer composer for example